application/backend/src/main/resources/apps/luna/components/accordion/accordionitem/.content.json

```diff
@@ -0,0 +1,7 @@
+{
+  "isContainer": false,
+  "allowedComponents": ["/apps/luna/components/accordion/accordioncontent"],
```

Comment:
You can remove these two lines. It's not a container by default and doesn't need allowedComponents - they are added only in initial content, and there is no chance to add them manually.